index 609b28a..83554b4 100644
--- a/src/emitter.c
+++ b/src/emitter.c
@@ -663,14 +663,14 @@ yaml_emitter_emit_document_start(yaml_emitter_t *emitter,
          * This can happen if a block scalar with trailing empty lines
          * is at the end of the stream
          */
-        if (emitter->open_ended == 2)
-        {
-            if (!yaml_emitter_write_indicator(emitter, "...", 1, 0, 0))
-                return 0;
-            emitter->open_ended = 0;
-            if (!yaml_emitter_write_indent(emitter))
-                return 0;
-        }
+//        if (emitter->open_ended == 2)
+//        {
+//            if (!yaml_emitter_write_indicator(emitter, "...", 1, 0, 0))
+//                return 0;
+//            emitter->open_ended = 0;
+//            if (!yaml_emitter_write_indent(emitter))
+//                return 0;
+//        }
         if (!yaml_emitter_flush(emitter))
             return 0;
