diff --git a/ngx_http_upload_module.c b/ngx_http_upload_module.c
index 93ded70..e7be6f5 100644
--- a/ngx_http_upload_module.c
+++ b/ngx_http_upload_module.c
@@ -8,23 +8,13 @@
 #include <ngx_http.h>
 #include <nginx.h>

-#if (NGX_HAVE_OPENSSL_MD5_H)
 #include <openssl/md5.h>
-#else
-#include <md5.h>
-#endif

-#if (NGX_OPENSSL_MD5)
 #define  MD5Init    MD5_Init
 #define  MD5Update  MD5_Update
 #define  MD5Final   MD5_Final
-#endif

-#if (NGX_HAVE_OPENSSL_SHA1_H)
 #include <openssl/sha.h>
-#else
-#include <sha.h>
-#endif

 #define MULTIPART_FORM_DATA_STRING              "multipart/form-data"
 #define BOUNDARY_STRING                         "boundary="
