diff -rupN rubygems-1.8.24-orig/lib/rubygems/specification.rb rubygems-1.8.24/lib/rubygems/specification.rb
--- rubygems-1.8.24-orig/lib/rubygems/specification.rb	2012-04-27 22:54:03.000000000 +0000
+++ rubygems-1.8.24/lib/rubygems/specification.rb	2012-11-17 00:15:54.000000000 +0000
@@ -1009,8 +1009,13 @@ class Gem::Specification
               elsif /\A(\d{4})-(\d{2})-(\d{2}) \d{2}:\d{2}:\d{2}\.\d+?Z\z/ =~ date then
                 Time.utc($1.to_i, $2.to_i, $3.to_i)
               else
-                raise(Gem::InvalidSpecificationException,
-                      "invalid date format in specification: #{date.inspect}")
+                begin
+                  require 'time'
+                  Time.parse date
+                rescue
+                  raise(Gem::InvalidSpecificationException,
+                        "invalid date format in specification: #{date.inspect}")
+                end
               end
             when Time, Date then
               Time.utc(date.year, date.month, date.day)